packages/@react-spectrum/s2/src/SelectBox.tsx

```diff
@@ -0,0 +1,379 @@
+    isDisabled: 0.4
+  }
+});
+
+const textContainer = style({
+  display: 'flex',
+  flexDirection: {
+    orientation: {
+      horizontal: 'column'
+    }
+  },
+  justifyContent: 'center',
+  alignItems: {
+    default: 'center',
+    orientation: {
+      horizontal: 'start'
+    }
+  },
+  gap: {
+    default: 12,
+    orientation: {
+      horizontal: 2
+    }
+  },
+  flex: {
+    orientation: {
+      horizontal: '1 0 0'
+    }
+  },
+  width: '100%',
+  minWidth: 0,
+  color: {
+    default: 'neutral',
+    isDisabled: 'disabled'
+  },
+}, getAllowedOverrides());
+
+const descriptionText = style({
+  alignSelf: 'stretch', 
+  width: '100%',
+  minWidth: 0,
+  overflow: 'hidden',
+  maxHeight: {
+    orientation: {
+      horizontal: 120,
+    }
+  },
+  color: {
+    default: 'neutral',
+    isDisabled: 'disabled'
+  }
+});
+
+const labelText = style({
+  display: 'block',
+  width: '100%',
+  overflow: 'hidden',
+  minWidth: 0,
+  textAlign: {
+    default: 'center',
+    orientation: {
+      horizontal: 'start'
+    }
+  },
+  whiteSpace: 'nowrap',
+  textOverflow: 'ellipsis',
+  alignSelf: {
+    orientation: {
+      horizontal: 'stretch'
+    }
+  },
+  fontWeight: {
+    orientation: {
+      horizontal: 'bold'
+    }
+  },
+  color: {
+    default: 'neutral',
+    isDisabled: 'disabled'
+  }
+});
+
+
+const SelectBoxRenderPropsContext = createContext<{
+  isHovered?: boolean,
+  isFocusVisible?: boolean,
+  isPressed?: boolean
+}>({});
+
+/**
+ * SelectBox components allow users to select options from a list.
+ * Works as content within a ListBoxItem.
+ */
+export const SelectBox = /*#__PURE__*/ forwardRef(function SelectBox(props: SelectBoxProps, ref: FocusableRef<HTMLDivElement>) {
+  [props, ref] = useSpectrumContextProps(props, ref, SelectBoxSpectrumContext);
+  let {
+    children, 
+    value, 
+    isDisabled: individualDisabled = false, 
+    UNSAFE_style
+  } = props;
+  let divRef = useRef<HTMLDivElement | null>(null);
+  let domRef = useFocusableRef(ref, divRef);
+  
+  let contextValue = useContext(SelectBoxContext);
+  let {
+    orientation = 'vertical',
+    selectedKeys,
+    isDisabled: groupDisabled = false,
+    isCheckboxSelection = false
+  } = contextValue;
+
+  let renderProps = useContext(SelectBoxRenderPropsContext);
+
+  const size = 'M';
+  const isDisabled = individualDisabled || groupDisabled;
+  const isSelected = selectedKeys === 'all' || (selectedKeys && selectedKeys.has(value));
+  
+  const childrenArray = React.Children.toArray(children);
+  const illustrationSlot = childrenArray.find((child: any) => child?.props?.slot === 'illustration');
+  const textSlot = childrenArray.find((child: any) => child?.props?.slot === 'text');
+  const descriptionSlot = childrenArray.find((child: any) => child?.props?.slot === 'description');
+  const otherChildren = childrenArray.filter((child: any) => 
+    !['illustration', 'text', 'description'].includes(child?.props?.slot)
+  );
+
+  return (
+    <div
```

Comment:
if you are able to make `SelectBox` into a `ListBoxItem`, you can reference `ComboBoxItem`. and if there's a reason that this wouldn't work that i might not be aware of, feel free to pushback

https://github.com/adobe/react-spectrum/blob/2ee02a0afccfcbb4611e3f4085739ce80852d133/packages/%40react-spectrum/s2/src/ComboBox.tsx#L373-L410